pnpm-workspace.yaml

```diff
@@ -6,6 +6,6 @@ packages:
 catalog:
   '@cloudflare/workers-types': ^4.20240919.0
   'typescript': ^5.6.2
-  'vite': '6.0.0-beta.2'
+  'vite': "https://pkg.pr.new/vite@main"
```

Comment:
Instead of using the main prerelease we can now use the latest 6 beta release: [`6.0.0-beta.4`](https://github.com/vitejs/vite/blob/v6.0.0-beta.4/packages/vite/CHANGELOG.md#600-beta4-2024-10-23)

or we can keep using the main prerelease to try to closely keep up to date with vite, whatever you guys prefer is fine by me

Reply:
Cool! Yes, let's go with beta.4

Reply:
```suggestion
  'vite': '6.0.0-beta.4'
```